dev-ml/opam/opam-2.0.10-r1.ebuild

```diff
@@ -22,7 +22,8 @@ RDEPEND="
 	dev-ml/extlib:=
 	~dev-ml/opam-client-${PV}:=
 	dev-ml/opam-file-format:=
-	dev-ml/re:="
+	dev-ml/re:=
+	sys-apps/bubblewrap:="
```

Comment:
Operator `:=` for `bubblewrap` is probably unnecessary (it does not provide any shared libraries).

Reply:
You're probably right, I really don't know that much about Ebuilds, I just wanted to help hehe.
@xgqt After removing `:=` should I open a new PR and close this one?

Reply:
> You're probably right, I really don't know that much about Ebuilds, I just wanted to help hehe. @xgqt After removing `:=` should I open a new PR and close this one?

Oh, no please do not re-open, force push to your branch. :)